cx2t/src/claw/wani/transformation/sca/ParallelizeForward.java

```diff
@@ -561,14 +563,17 @@ private void transformStd(XcodeProgram xcodeml, Translator translator)
     propagatePromotion(xcodeml, (ClawTranslator) translator);
 
     Xnode exprStmt = _fctCall.matchAncestor(Xcode.EXPR_STATEMENT);
-    if(_claw.hasCreateClause()) {
+
+    // Peclat : Only useful for GPU
```

Comment:
We have your name already in the commit so just put a single comment without name is fine.